Sources/PackageDescription/PackageDependency.swift

```diff
@@ -222,14 +222,16 @@ extension Package.Dependency {
         return .init(name: nil, url: path, requirement: ._localPackageItem)
     }
 
-    /// Add a dependency to a local package on the filesystem.
+    /// Adds a package dependency to a local package on the filesystem.
     ///
-    /// The Swift Package Manager uses the package dependency as-is
-    /// and does not perform any source control access. Local package dependencies
```

Comment:
just curious: why prefer one over the other? 🤔

Reply:
Using contractions follows the Apple style guide.

> “As part of Apple’s informal voice, contractions are used and recommended throughout most documentation, interface text, and marketing copy. [...]”